test/kafka-exactly-once/mzcompose.yml

```diff
@@ -76,7 +76,7 @@ services:
       --data-directory=/share/mzdata
       -w1
       --disable-telemetry
-      --logical-compaction-window=off
+      --logical-compaction-window=1ms
```

Comment:
nit: i don't think you need to specify this because its the default

Reply:
I wanted to make it explicit, in case the default ever changes. But I'm happy to remove.